### DISCUSSION TITLE
hide sensitive data when capturing response

### DISCUSSION THREAD

**Discussion by ariaieboy** [saloonphp/saloon#222] [Q&A]
Hi, I am using this package to rewrite my SDK for Caprover https://github.com/ariaieboy/caprover-sdk 
And I want to know if there is any way to hide sensitive data or replace it with fake ones when we use Recording Responses for testing purposes.

**Comment by Sammyjo20** [comment]
Hey @ariaieboy apologies for the delay to your question - we're currently in discussion about this feature in PR #234 - I can't promise it will be released but it would be great to have any suggestions/feedback here.
